### PR TITLE
TST: Update Travis CI environment -- pysat 3.0.0 edition

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses #383.  Twin to #384.

## Type of change

- Bug fix -- testing environment (non-breaking change which fixes an issue)

# How Has This Been Tested?

via Travis CI

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,17 @@
...
    - Decreased time to load COSMIC GPS data by about 50%
    - Added DE2 Langmuir Probe, NACS, RPA, and WATS instruments
    - Updated `test_files.py` to be pytest compatible
+   - Changed madrigal methods to use `madrigalWeb` as a module rather than
+     calling it externally
+- Deprecation Warning
+  - custom.add will be renamed custom.attach in pysat 3.0.0
 - Documentation
-  - Fixed description of tag and sat_id behaviour in testing instruments
+   - Fixed description of tag and sat_id behaviour in testing instruments
```
Comment (aburrell) [inline CHANGELOG.md]: Should this space be here?

Comment (jklenzing) [reply CHANGELOG.md]: Nope.

**Comment by jklenzing** [comment]
Deprecation warnings and deprecated functions have been removed.

**Comment by jklenzing** [comment]
The plot thickens!  netcdf4 1.5.3 is the only version that supports python 3.8.  Meaning we cannot employ this temporary solution and test for python 3.8.

This should go on the agenda for Wednesday.

**Comment by jklenzing** [comment]
What should we do about the 3.8 tests?  Remove these?

**Pull request merged**